### PR TITLE
every package with plugin.xml should have a test for it

### PR DESCRIPTION
**Pull request by elharo**

### CODE REVIEW

**Comment by chanseokoh** [comment]
There are packages that don't have `plugin.xml`, and that's why the tests are failing.

**Comment by elharo** [comment]
PTAL @chanseokoh

File: plugins/com.google.cloud.tools.eclipse.sdk.test/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -11,7 +11,8 @@ Require-Bundle: org.junit,
  org.eclipse.jface,
  org.eclipse.equinox.common;bundle-version="3.7.0",
  com.google.cloud.tools.eclipse.util
-Import-Package: com.google.cloud.tools.eclipse.test.util.reflection,
+Import-Package: com.google.cloud.tools.eclipse.test.util,
```
Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.sdk.test/META-INF/MANIFEST.MF]: I think this should be reverted for `.sdk.test`.

File: plugins/com.google.cloud.tools.eclipse.usagetracker.test/META-INF/MANIFEST.MF [hunk 2]
```diff
@@ -10,3 +10,4 @@ Require-Bundle: org.junit,
  org.eclipse.equinox.preferences,
  com.google.cloud.tools.eclipse.test.dependencies;bundle-version="0.1.0",
  org.eclipse.swt
+Import-Package: com.google.cloud.tools.eclipse.test.util
```
Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.usagetracker.test/META-INF/MANIFEST.MF]: I think this should be reverted for `.usagetracker.test`.

File: plugins/com.google.cloud.tools.eclipse.sdk.test/META-INF/MANIFEST.MF (hunk 1 above)
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.sdk.test/META-INF/MANIFEST.MF]: done

File: plugins/com.google.cloud.tools.eclipse.usagetracker.test/META-INF/MANIFEST.MF (hunk 2 above)
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.usagetracker.test/META-INF/MANIFEST.MF]: done

File: plugins/com.google.cloud.tools.eclipse.util/plugin.properties [hunk 3]
```diff
@@ -1,2 +0,0 @@
-pluginName=Cloud Tools for Eclipse shared utility code
```
Comment (briandealwis) [inline plugins/com.google.cloud.tools.eclipse.util/plugin.properties]: And we don't want to externalize these strings?

Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.util/plugin.properties]: These weren't actually used.

**Pull request merged**